src/codegen/codegen_neuron_cpp_visitor.cpp

```diff
@@ -453,6 +453,16 @@ void CodegenNeuronCppVisitor::print_global_function_common_code(BlockType type,
     return;
 }
 
+void CodegenNeuronCppVisitor::print_nrn_init() {
+    printer->add_line(
+        "void nrn_init(_nrn_model_sorted_token const&, NrnThread*, Memb_list*, int) {}");
```

Comment:
```suggestion
    printer->fmt_line("static void {}(_nrn_model_sorted_token const& _sorted_token, NrnThread* _nt, Memb_list* _ml_arg, int _type) {{}}", method_name(naming::NRN_INIT_METHOD));
```
A simple suggestion for improvement at the moment. Same for the other functions as well

Reply:
Thank you, might be useful in the next PR. First let's get it as far as loading.